types/react/index.d.ts

```diff
@@ -2546,7 +2547,7 @@ declare namespace React {
         hr: DetailedHTMLFactory<HTMLAttributes<HTMLHRElement>, HTMLHRElement>;
         html: DetailedHTMLFactory<HtmlHTMLAttributes<HTMLHtmlElement>, HTMLHtmlElement>;
         i: DetailedHTMLFactory<HTMLAttributes<HTMLElement>, HTMLElement>;
-        iframe: DetailedHTMLFactory<IframeHTMLAttributes<HTMLIFrameElement>, HTMLIFrameElement>;
+        iframe: DetailedHTMLFactory<IframeHTMLAttributes<HTMLElement>, HTMLIFrameElement>;
```

Comment:
Doesn't look like this change belongs here?

Reply:
Removed.